src/pages/AnnualReport/AnnualReportTable/ClubAnnualReportInformation/ClubAnnualReportInformation.tsx

```diff
@@ -52,18 +52,19 @@ const ClubAnnualReportInformation = () => {
         setIsLoading(true);
         try {
             let clubReport = await getClubAnnualReportById(id);
+            console.log(clubReport);
```

Comment:
please delete console logs here and below

Reply:
done